### ISSUE TITLE
As an Xpress scientist, I need to access my proposal outside of the proposal software, so that I can schedule experiments

### ISSUE THREAD

**Issue by simonfernandes** [UserOfficeProject/user-office-project-issue-tracker#1029]
Xpress proposals submitted in the UOP should be scheduled, so there needs to be a way for proposals to flow from the UOP submission system into the Scheduler. This will be done via Allocations, which will ingest the proposals and which the Scheduler will query.

The point at which an Xpress proposal should be ingested is when the Xpress proposal reaches the `Approved` status. An Xpress proposal cannot reached `Approved` without an instrument assigned to it.

Allocations issue: https://github.com/isisbusapps/proposal-allocations/issues/557
Scheduler issue: Waiting on the Allocations work to complete before assessing whether any work needs to be done (@go994 aware of this)

**Comment by joshhdawes** [comment]
@simonfernandes @ACLay asks if this issue just covers the work that needs to be done by R&A? If so, it's blocked by the work being done by Submissions.

**Comment by simonfernandes** [comment]
@joshhdawes Hi, it was intended to capture the flow of data from submissions -> allocations -> scheduler. Would it be possible to create a separate issue in Allocations (sorry!) and then I can link them together here? It sounds correct that you're currently blocked and require more info/impelementation on our side.

**Comment by joshhdawes** [comment]
@simonfernandes sure! @ACLay could you please set the issue up? I think you can write something far more detailed, and accurate, than I can :)

Also @simonfernandes, can we agree on deadlines for this work? If I've understood correctly, your work on Submissions will need to be done first - could you let us know when you expect to be done by, so we can set a deadline on our side?

**Comment by ACLay** [comment]
Support xpress proposals from the UOP: https://github.com/isisbusapps/proposal-allocations/issues/557
Import legacy xpress proposals: https://github.com/isisbusapps/proposal-allocations/issues/558

**Comment by simonfernandes** [comment]
Our deadline for Xpress is the end of the year but I don't think you'll need everything to be done and dusted on our side before it can be hooked up to allocations, so long as you have the information of how it will work and a way to test. As yet we don't all of those answers as we're still defining the post-submission process.

We will also need to ensure these Xpress experiments can be scheduled via allocations.

@ACLay Will the legacy import be needed before we release Xpress?

**Comment by ACLay** [comment]
Nah, it just seemed like something we should do eventually for consistency

**Comment by ACLay** [comment]
I say eventually, hopefully it won't take too long. I don't want to have to be rewriting all the reporting tool queries multiple times to be hybrid and then just pointing at one new system.

**Comment by ACLay** [comment]
Reporting tool issue: https://github.com/isisbusapps/reporting-tool/issues/845

**Comment by joshhdawes** [comment]
> so long as you have the information of how it will work and a way to test. As yet we don't all of those answers as we're still defining the post-submission process.
> 
> We will also need to ensure these Xpress experiments can be scheduled via allocations.

@simonfernandes @ACLay it sounds to me like we can't start on this yet, then, if we're waiting for things to be defined a bit more.

I'd like to put this onto our sprint board as an epic, so we can attach a start and end date to the work (which we may only be able to do once you guys have defined things a bit more).

**Comment by ACLay** [comment]
> I'd like to put this onto our sprint board as an epic, so we can attach a start and end date to the work (which we may only be able to do once you guys have defined things a bit more).

Sounds good to me

**Comment by joshhdawes** [comment]
@ACLay I've assigned this to you for the time being (to do the work on the R&A side, of course), but we can consider giving the work to someone else (or multiple people)

**Comment by simonfernandes** [comment]
@ACLay Can this be closed off now?

Edit: I see this is an epic actually, got confused with these!
